### PR TITLE
[TS-166] example code for packet of workers

### PR DESCRIPTION
**Pull request by jenchik**

### CODE REVIEW

**Review by im-kulikov — COMMENTED** [review]
giphy-5

Just use English

File: examples/workers/main.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+)
+
+type myDelayLock struct {
+	stop bool
+}
+
+func (m *myDelayLock) Next(t time.Time) time.Time {
+	if m.stop {
+		fmt.Printf("[%s] instance shell be stopped\n", time.Now().Format("15:04:05"))
+		// отправляя нулевое время, мы по факту выключаем воркер,
```
Comment (kpeo) [inline examples/workers/main.go]: translate it to english please

**Comment by im-kulikov** [comment]
@kpeo pls, help @jenchik with translation..

File: examples/workers/main.go (hunk 1 above)
Comment (pcherednichenko) [reply examples/workers/main.go]: I do not speak Russian, what does this mean?

Comment (im-kulikov) [reply examples/workers/main.go]: @pcherednichenko Судячи з вашого логіну, ви говорите українською? (comment writen with Google Translate)

File: examples/workers/main.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	"go.uber.org/atomic"
+)
+
+type myDelayLock struct {
+	stop bool
+}
+
+func (m *myDelayLock) Next(t time.Time) time.Time {
+	if m.stop {
+		fmt.Printf("[%s] instance shell be stopped\n", time.Now().Format("15:04:05"))
```
Comment (kpeo) [inline examples/workers/main.go]: will be stopped

File: examples/workers/main.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+	// worker will run as custom scheduler
+	// example of scheduler (using workers.Schedule interface)
+	delay := new(myDelayLock)
+	err = workers.New(workers.Options{
+		Name:     "worker #4",
+		Schedule: delay,
+		Handler: func() {
+			if step.Load() > 4 && !delay.stop {
+				fmt.Printf("[%s] worker #4: send command exit\n", time.Now().Format("15:04:05"))
```
Comment (kpeo) [inline examples/workers/main.go]: send command 'exit'

**Review by kpeo — APPROVED** [review]

**Review by pcherednichenko — APPROVED** [review]

**Pull request merged**